packages/utils/pack-up/tsconfig.json

```diff
@@ -1,5 +1,9 @@
 {
   "extends": "tsconfig/base.json",
   "include": ["src"],
-  "exclude": ["node_modules"]
+  "exclude": ["node_modules"],
+  "compilerOptions": {
+    "module": "ESNext",
+    "moduleResolution": "Bundler"
+  }
```

Comment:
We updated the base config – I don't remember seeing a PR for this 🙃 

Rebasing broke the feature branch, but realistically I think this is the correct settings because we're using a Bundler to bundle the code not `tsc`.